util/src/test/scala/io/guanaco/alerta/util/AlertaRouteBuilderSupportTest.scala

```diff
@@ -147,13 +166,34 @@ class AlertaRouteBuilderSupportTest extends CamelTestSupport with AlertaCamelTes
               .when(simple("${body} contains 'broken'"))
                 .setHeader(ATTRIBUTES_HEADER, constant(AttributesDynamic))
                 .throwException(new IllegalStateException("It's broken! It really is broken!!"))
+
+              .when(simple("${body} contains 'Initial'"))
+                .setHeader(ATTRIBUTES_HEADER, constant(AttributesDynamic))
+                .throwException(new IllegalStateException("Initial error"))
+                .setBody(constant("Another message"))
+                .throwException(new IllegalStateException("Another error"))
+                .setBody(constant("Yet another message"))
+
               .otherwise()
                 .to(END)
           // format: on
         }
       }
     )
 
+  case class ExceptionBean() {
```

Comment:
Ik snap wat de bedoeling is van deze `ExceptionBean` maar ik zie precies niet zo dadelijk waar die gebruikt wordt? 

Hoort die niet ergens in die `.when(simple("${body} contains 'Initial'"))` branch van de route gebruikt te worden?